Assets/Vimeo/Scripts/Recorder/VimeoRecorder.cs

```diff
@@ -51,7 +51,11 @@ public void EndRecording()
             isRecording = false;
             encoder.EndRecording();
 
-            PublishVideo(encoder.GetVideoFilePath());
+            if (autoUpload) {
+                PublishVideo();
+            } else {
+                Debug.Log("[VimeoRecorder] You set autoUpload to false, you can easily use the vimeoPlayer.PublishVideo() to easily trigger the upload process");
```

Comment:
Perhaps a simpler way to put it..
Debug.Log("[Vimeo] Video did not automatically upload. VimeoPlayer.autoUpload is set to false.");

Reply:
Also, since this is new logic, then there should be a corresponding test to validate that this warning is thrown.

Reply:
I validate using the booleans - `isRecording`, `isUploading` and `autoUpload` - I think we shouldn't relay on logs being printed since we might get rid of them at some point and it will be hard to follow